src/frontends/common_translators/include/common_translators.hpp

```diff
@@ -19,6 +19,8 @@ COMMON_OP_CONVERTER(translate_imag);
 COMMON_OP_CONVERTER(translate_atan2);
 COMMON_OP_CONVERTER(translate_angle);
 
+COMMON_OP_CONVERTER(translate_equal_op);
```

Comment:
```suggestion
COMMON_OP_CONVERTER(translate_equal);
```

let us have names without `_op` suffix. It is obvious that this is for ops:)